pkg/ns/ns_linux.go

```diff
@@ -1,3 +1,5 @@
+// +build linux
```

Comment:
Since this file already ends with `_linux.go`, adding the build constraint comment is superfluous.  See [docs here](https://golang.org/pkg/go/build/#hdr-Build_Constraints), specifically:
> If a file's name, after stripping the extension and a possible _test suffix, matches any of the following patterns:
> ```
> *_GOOS
> *_GOARCH
> *_GOOS_GOARCH
> ```
> (example: source_windows_amd64.go) where GOOS and GOARCH represent any known operating system and architecture values respectively, then the file is considered to have an implicit build constraint requiring those terms (in addition to any explicit constraints in the file).

Reply:
true. I was just being explicit. I can remove the build tag.